acceptance/tests/puppetserver_requests.rb

```diff
@@ -0,0 +1,26 @@
+require 'beaker-http'
+
+test_name 'Ensure that requests can be built out to the puppetserver'
+
+step 'install latest released puppet agent' do
+  install_puppet_agent_on(hosts)
+end
+
+step 'install the latest puppet-server on the master' do
+  install_package(master, 'puppetserver')
+  on master, 'service puppetserver start'
+end
+
+step 'generate a new beaker http connection object'
+beaker_http_connection = Beaker::Http::Connection.new(master)
+
+step 'configure the connection to connect to the master port'
+beaker_http_connection.connection.port = 8140
```

Comment:
I didn't want to overload `def_delegators` with too many of the Faraday methods, but it seems useful to add these simple setters like port, scheme, path, and host. I think you are right, it makes sense to delegate those.